app/components/team_sidebar/team_list/team_item/index.ts

```diff
@@ -21,9 +20,14 @@ type WithTeamsArgs = WithDatabaseArgs & {
 
 const enhance = withObservables(['myTeam'], ({myTeam, database}: WithTeamsArgs) => {
     const myChannels = queryMyChannelsByTeam(database, myTeam.id).observeWithColumns(['mentions_count', 'is_unread']);
+    const notifyProps = observeAllMyChannelNotifyProps(database);
     const hasUnreads = myChannels.pipe(
+        combineLatestWith(notifyProps),
         // eslint-disable-next-line max-nested-callbacks
-        switchMap((val) => of$(val.reduce((acc, v) => acc || v.isUnread, false))),
+        map(([mycs, notify]) => mycs.reduce((acc, v) => {
+            const isMuted = notify?.[v.id]?.mark_unread === 'mention';
+            return acc || (v.isUnread && !isMuted);
```

Comment:
should we take into account the mentions if a channel is muted?
something like:

```suggestion
            const isMuted = notify?.[v.id]?.mark_unread === 'mention';
            return acc || (isMuted ? v.hasMentions : v.isUnread));
```

Reply:
@koox00 this is really about unreads, mentions are handled on its own observer, see line 36